### PR TITLE
Fixed scripts

### PR DESCRIPTION
**Pull request by realsystem**
clean volumes works fine, define WORKSPACE for non-jenkins run and adde check for default lab usage

### CODE REVIEW

File: devlab/utils/os_cli.sh [hunk 1]
```diff
@@ -146,7 +150,7 @@ function delete_volume {
 }
 
 function get_volume_list {
-  ${ssh} nova volume-list --all-tenants | grep none | cut -d ' ' -f 2
+  ${ssh} nova volume-list --all-tenants | grep available | cut -d ' ' -f 2
```
Comment (roman-verchikov) [inline devlab/utils/os_cli.sh]: This skips `in-use` volumes, is this expected?... I'm assuming this function would return *all* volumes, so perhaps `cinder list --all-tenants | sed -nr 's/^\| (\w{8}-\w{4}-\w{4}-\w{4}-\w{12}).*/\1/p'` would do the job correctly.

File: devlab/utils/os_cli.sh [hunk 2]
```diff
@@ -146,7 +150,8 @@ function delete_volume {
 }
 
 function get_volume_list {
-  ${ssh} nova volume-list --all-tenants | grep none | cut -d ' ' -f 2
+  ${ssh} cinder list --all-tenants | sed -nr 's/^\| (\w{8}-\w{4}-\w{4}-\w{4}-\w{12}).*/\1/p'
+#  ${ssh} nova volume-list --all-tenants | grep available | cut -d ' ' -f 2
```
Comment (roman-verchikov) [inline devlab/utils/os_cli.sh]: Do we really need this comment?.. Doesn't seem to have much value...

**Comment by roman-verchikov** [comment]
please rebase

**Comment by roman-verchikov** [comment]
please rebase

**Comment by realsystem** [comment]
ready for review/merge

**Comment by miarmak** [comment]
Do we still need this patch?

**Comment by realsystem** [comment]
Sure, but need to fix issue in devlab SAIC-445 and re make this patch

File: devlab/utils/clean_run.sh [hunk 3]
```diff
@@ -55,5 +55,5 @@ ssh root@$DST \
 delete from cinder.volumes;delete from cinder.reservations;delete from cinder.quota_usages;\""
 
 cd $cur_pwd
-bash devlab/utils/os_cli.sh clean dst
-fab migrate:configuration.ini
+bash devlab/utils/os_cli.sh clean dst grizzly icehouse
```
Comment (ntymtsiv) [inline devlab/utils/clean_run.sh]: Why we still use icehouse as destination cloud instead of juno?

**Comment by roman-verchikov** [comment]
+1, can be merged.

**Pull request merged**